.rubocop.yml

```diff
@@ -16,6 +16,7 @@ Rails/TimeZone:
 RSpec/MultipleExpectations:
   Exclude:
     - 'spec/features/**/*'
+    - 'spec/models/user_spec.rb'
```

Comment:
Is it team opinion that multiple expectations are good (or at least acceptable) across the board? If so, I want to flag this rule for removal in `bixby`.

Either way, I'm fine with the relevant specs here for now.

Reply:
I think multiple expectations are fine and even desirable, especially in our context where test setup can be expensive and we want things to run faster. I'd be very happy to see this removed from bixby.

Reply:
:+1: https://github.com/samvera-labs/bixby/issues/19